### ISSUE TITLE
Numpydoc docstring: last line in docstring should be empty and first line after docstring shold be code

### ISSUE THREAD

**Issue by PeterFogh** [NilsJPWerner/autoDocstring#61]
Hi, first of all, thanks for the useful VS Code extension.

I mainly use the Numpy docstring conventions, and after a first glance, I found the extension do not fully follow [the Numpydoc conventions](https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard) with respect to that the last line in docstring should be empty and the first line after docstring should be code.

Hence, the following example:
```
def fun(a, b=1):
    """
    c = 1
    d = 'hello'
    return c, d
```
shold result in: 
```
def fun(a, b=1):
    """
    [summary]

    [description]

    Parameters
    ----------
    a : [type]
        [description]
    b : int, optional
        [description] (the default is 1, which [default_description])

    Returns
    -------
    [type]
        [description]

    """
    c = 1
    d = 'hello'
    return c, d
```

but the extension outputs:
```
def fun(a, b=1):
    """
    [summary]
    
    [description]
    
    Parameters
    ----------
    a : [type]
        [description]
    b : int, optional
        [description] (the default is 1, which [default_description])
    
    Returns
    -------
    [type]
        [description]
    """

    c = 1
    d = 'hello'
    return c, d
```

**Comment by ali80** [comment]
yes, it also doesn't comply with `google style python docstrings`
[https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html)
there is an extra empty line after the docstring ending

**Comment by NilsJPWerner** [comment]
Fixed in the version 0.3.0
